tutorial/README.md

```diff
@@ -0,0 +1,42 @@
+# Why Helm?
+
+[Helm](https://docs.helm.sh/) is often described as the Kubernetes application package manager. So, what does Helm give you over using Kubernetes directly?
+
+# Objectives
+
+These labs provide an insight on the advantages of using Helm instead of using Kubernetes directly through `kubectl`. By the end of this course, you'll:
+* Understand the core concepts of Helm
+* Understand the advantages of deployment using Helm over Kubernetes directly, looking at:
+  * Application management
+  * Updates
+  * Configuration
+  * Revision management
+  * Repositories and chart sharing
+
+# Prerequesites
+
+* Have a running Kubernetes cluster. See the [IBM Cloud Kubernetes Service](https://console.bluemix.net/docs/containers/cs_tutorials.html) or [Kubernetes Getting Started Guide](https://kubernetes.io/docs/setup/) for details about creating a cluster.
```

Comment:
Please use `https://cloud.ibm.com/docs/containers/cs_tutorials.html#cs_cluster_tutorial` instead of the console.bluemix.net url.

Reply:
done